docs/source/devguides/walkthroughs/DrivenMatlabAgent-Walkthrough.rst

```diff
@@ -72,35 +75,118 @@ Installation:
 
     Follow link: http://volttron.readthedocs.io/en/develop/install.html
 
-2. Add subtree volttron-applications under volttron/applications by using
-   the following command:
+2. Clone volttron-applications in any directory outside of your volttron
+   source directory. (Best practise: clone in the parent directory of volttron
+   source. For example, /users/user1/volttron and
+   /users/user1/volttron-applications):
 
 .. code-block:: console
 
-    git subtree add --prefix applications https://github.com/VOLTTRON/volttron-applications.git develop --squash
+    git clone https://github.com/VOLTTRON/volttron-applications.git
 
-Configuration
-~~~~~~~~~~~~~
+Install Dependencies
+~~~~~~~~~~~~~~~~~~~~
+
+To test the interaction between matlab driven agent and matlab code you would
+ need the following agents installed and running
+
+1. Run master driver and configure it to publish device data. Data published
```

Comment:
Change to be MasterDriver 

Say something about the topics published by master driver need to map onto the driven mat lab agent.